CHANGES.rst

```diff
@@ -92,9 +94,10 @@ Bugs
 
 #. Units and Quantities were sometimes failing. Also they were omitted from documentation.
 #. Better handling of ``Infinite`` quantities.
-#. Fix ``Precision`` compatibility with WMA.
+#. Fix ``Precision`` and ``Accuracy``compatibility with WMA. In particular, ``Precision[0.]`` and ``Accuracy[0.]``
+#. Accuracy in numbers usint the notation ``` n.nnn``acc ```  now is properly handled.
```

Comment:
usint -> using

Reply:
A small thing about using the word "Fix". Does it mean that all the problems with Precision have really been corrected or is it that the previous problems have been reduced, that is, that Precision has been _improved_? 

The section on "Precision" says

> This is rather a proof-of-concept than a full implementation

Is that still the case after this PR? 

The section on "Accuracy" says: 

> Notice that the value is not exactly equal to the [value] obtained in WMA

Is this still accurate?

Reply:
In all the cases I have tested, I get the same result as in WMA (up to small numeric differences due to the mpmath implementation).

Reply:
The differences I found are in this order:

```'Accuracy[0.32`30*^30]``` 

WMA:      0.4948500216800894
Mathics: 0.49485002168009373

For Complex, there could be more differences:

```Precision[12`20*^30+13`15*^24]```
WMA:       19.9553
Mathics:   20.

```Accuracy[12`20*^30+13`15*^24]```
WMA:       -11.1239
Mathics:  -11.0792

Probably in this case the normalization is different. In any case, differences are of at most ~O(1/10). So, all the currently detected issues are fixed. But OK, I can rephrase it.

Reply:
Thanks for performing the experiments. It is nice to know this. I think we should document this somewhere so we don't lose this useful information.